domain/services/model.go

```diff
@@ -174,7 +179,6 @@ func (s *ModelServices) Application() *applicationservice.WatchableService {
 		applicationstate.NewState(changestream.NewTxnRunnerFactory(s.modelDB), s.clock, log),
 		secretstate.NewState(changestream.NewTxnRunnerFactory(s.modelDB), log),
 		s.storageRegistry,
-		resource.NewResourceStoreFactory(s.objectstore),
```

Comment:
Looks like the moved resource store has not been wired up into the application service. Has this been omitted on purpose?

Reply:
Yes, please see the PR description.

Reply:
I don't see anything in there mentioning this unwiring, I looked there initially.